web/packages/teleport/src/Main/Main.tsx

```diff
@@ -319,7 +319,7 @@ export const ContentMinWidth = ({ children }: { children: ReactNode }) => {
           overflow-y: auto;
           ${marginTransitionCss({
             sidePanelOpened: infoGuideSidePanelOpened,
-            panelWidth,
+            panelWidth: infoGuideConfig?.viewHasOwnSidePanel ? 0 : panelWidth,
```

Comment:
does this need to have `px` after? im guessing not but just curious

Reply:
no, we just pass a number, and append px at the callsite